ledger-api/testing-utils/src/main/scala/com/digitalasset/ledger/api/testing/utils/AkkaBeforeAndAfterAll.scala

```diff
@@ -12,9 +12,9 @@ import scala.concurrent.duration._
 
 trait AkkaBeforeAndAfterAll extends BeforeAndAfterAll { self: Suite =>
 
-  protected implicit val system: ActorSystem = ActorSystem(this.getClass.getSimpleName)
+  protected implicit def system: ActorSystem = ActorSystem(this.getClass.getSimpleName)
```

Comment:
`val` to `def` would create a new ActorMaterializer every time, having a terrible outcome..